CI build returns the following error message. Posting it here as a friendly reference for others!
```
Traceback (most recent call last):
  File "/usr/lib/python3.6/unittest/case.py", line 59, in testPartExecutor
    yield
  File "/usr/lib/python3.6/unittest/case.py", line 605, in run
    testMethod()
  File "/usr/lib/python3.6/doctest.py", line 2199, in runTest
    raise self.failureException(self.format_failure(new.getvalue()))
AssertionError: Failed doctest test for tensorflow.python.keras.backend.random_binomial
  File "/b/f/w/bazel-out/k8-opt/bin/tensorflow/tools/docs/tf_doctest.runfiles/org_tensorflow/tensorflow/python/keras/backend.py", line 316, in random_binomial

----------------------------------------------------------------------
File "/b/f/w/bazel-out/k8-opt/bin/tensorflow/tools/docs/tf_doctest.runfiles/org_tensorflow/tensorflow/python/keras/backend.py", line 340, in tensorflow.python.keras.backend.random_binomial
Failed example:
    random_binomial_tensor = tf.keras.backend.random_binomial(shape=(2,3),
Exception raised:
    Traceback (most recent call last):
      File "/usr/lib/python3.6/doctest.py", line 1330, in __run
        compileflags, 1), test.globs)
      File "<doctest tensorflow.python.keras.backend.random_binomial[0]>", line 1
        random_binomial_tensor = tf.keras.backend.random_binomial(shape=(2,3),
                                                                             ^
    SyntaxError: unexpected EOF while parsing
----------------------------------------------------------------------
File "/b/f/w/bazel-out/k8-opt/bin/tensorflow/tools/docs/tf_doctest.runfiles/org_tensorflow/tensorflow/python/keras/backend.py", line 342, in tensorflow.python.keras.backend.random_binomial
Failed example:
    random_binomial_tensor
Exception raised:
    Traceback (most recent call last):
      File "/usr/lib/python3.6/doctest.py", line 1330, in __run
        compileflags, 1), test.globs)
      File "<doctest tensorflow.python.keras.backend.random_binomial[1]>", line 1, in <module>
        random_binomial_tensor
    NameError: name 'random_binomial_tensor' is not defined
```

--- a/tensorflow/python/keras/backend.py
+++ b/tensorflow/python/keras/backend.py
@@ -5659,7 +5659,7 @@ def random_normal(shape, mean=0.0, stddev=1.0, dtype=None, seed=None):
   Example:
 
   >>> random_normal_tensor = tf.keras.backend.random_normal(shape=(2,3),
-  mean=0.0, stddev=1.0)
+  ... mean=0.0, stddev=1.0)
   >>> random_normal_tensor
   <tf.Tensor: shape=(2, 3), dtype=float32, numpy=...,
   dtype=float32)>

--- a/tensorflow/python/keras/backend.py
+++ b/tensorflow/python/keras/backend.py
@@ -1480,7 +1480,7 @@ def random_uniform_variable(shape, low, high, dtype=None, name=None, seed=None):
   Example:
 
   >>> kvar = tf.keras.backend.random_uniform_variable(shape=(2,3),
-  low=0.0, high=1.0)
+  ... low=0.0, high=1.0)
   >>> kvar
   <tf.Variable 'Variable:0' shape=(2, 3) dtype=float32, numpy=...,
   dtype=float32)>
@@ -1515,7 +1515,7 @@ def random_normal_variable(shape, mean, scale, dtype=None, name=None,
   Example:
 
   >>> kvar = tf.keras.backend.random_normal_variable(shape=(2,3),
-  mean=0.0, scale=1.0)
+  ... mean=0.0, scale=1.0)
   >>> kvar
   <tf.Variable 'Variable:0' shape=(2, 3) dtype=float32, numpy=...,
   dtype=float32)>
@@ -5691,7 +5691,7 @@ def random_uniform(shape, minval=0.0, maxval=1.0, dtype=None, seed=None):
   Example:
 
   >>> random_uniform_tensor = tf.keras.backend.random_uniform(shape=(2,3),
-  minval=0.0, maxval=1.0)
+  ... minval=0.0, maxval=1.0)
   >>> random_uniform_tensor
   <tf.Tensor: shape=(2, 3), dtype=float32, numpy=...,
   dtype=float32)>
@@ -5724,7 +5724,7 @@ def random_binomial(shape, p=0.0, dtype=None, seed=None):
   Example:
 
   >>> random_binomial_tensor = tf.keras.backend.random_binomial(shape=(2,3),
-  p=0.5)
+  ... p=0.5)
   >>> random_binomial_tensor
   <tf.Tensor: shape=(2, 3), dtype=float32, numpy=...,
   dtype=float32)>

